engine/install/ubuntu.md

```diff
@@ -1,42 +1,43 @@
 ---
-description: Instructions for installing Docker Engine - Community on Ubuntu
+description: Instructions for installing Docker Engine on Ubuntu
 keywords: requirements, apt, installation, ubuntu, install, uninstall, upgrade, update
 redirect_from:
 - /engine/installation/ubuntulinux/
 - /installation/ubuntulinux/
+- /engine/installation/linux/ubuntu/
 - /engine/installation/linux/ubuntulinux/
 - /engine/installation/linux/docker-ce/ubuntu/
 - /install/linux/ubuntu/
-- /engine/installation/linux/ubuntu/
-title: Get Docker Engine - Community for Ubuntu
+- /install/linux/docker-ce/debian/
```

Comment:
I tried https://docs-stage.docker.com/install/linux/docker-ce/ubuntu/ but it gives me a 404.

Should it be?
```suggestion
- /install/linux/docker-ce/ubuntu/
```

Reply:
arf, yes, this one should be ubuntu; let me open a quick PR

Reply:
Maybe there are similar redirects, it was just the first one I tried.

Reply:
@StefanScherer opened https://github.com/docker/docker.github.io/pull/10583

Reply:
> Maybe there are similar redirects, it was just the first one I tried.

Thanks Stefan. Verified the redirects from `/install/linux/` and they all point to the right topics now under `/engine/`.